Formula/tremor-runtime.rb

```diff
@@ -0,0 +1,96 @@
+class TremorRuntime < Formula
+  desc "Early-stage event processing system for unstructured data"
+  homepage "https://www.tremor.rs/"
+  url "https://github.com/tremor-rs/tremor-runtime/archive/refs/tags/v0.12.4.tar.gz"
+  sha256 "91cbe0ca5c4adda14b8456652dfaa148df9878e09dd65ac6988bb781e3df52af"
+  license "Apache-2.0"
+  head "https://github.com/tremor-rs/tremor-runtime.git", branch: "main"
+
+  depends_on "cmake" => :build
+  depends_on "rust" => :build
+
+  on_linux do
+    depends_on "pkg-config" => :build
+    depends_on "gcc"
+    depends_on "llvm"
+    depends_on "openssl@1.1"
+  end
+
+  fails_with gcc: "5" # C++20
+
+  def install
+    inreplace ".cargo/config", "+avx,+avx2,", ""
+
+    system "cargo", "install", *std_cargo_args(path: "tremor-cli")
+
+    (bash_completion/"tremor").write Utils.safe_popen_read("#{bin}/tremor", "completions", "bash")
+    (zsh_completion/"_tremor").write Utils.safe_popen_read("#{bin}/tremor", "completions", "zsh")
+    (fish_completion/"tremor.fish").write Utils.safe_popen_read("#{bin}/tremor", "completions", "fish")
+
+    # main binary
+    bin.install "target/release/tremor"
+
+    # stdlib
+    (lib/"tremor-script").install (buildpath/"tremor-script/lib").children
+
+    # sample config for service
+    (etc/"tremor").install "docker/config/docker.troy" => "main.troy"
+
+    # wrapper
+    (bin/"tremor-wrapper").write_env_script (bin/"tremor"), TREMOR_PATH: "#{lib}/tremor-script"
+  end
+
+  # demo service
```

Comment:
this can be removed?

Reply:
I don't understand? Are you asking me to remove the wrapper, or the comment?

The wrapper is needed because the standard library path is hardcoded in the code and cannot be changed. The wrapper provides a way to use the binary with the appropriate environment to find the standard library.

This way the user has both the options - use the binary and provide the environment yourself, or use the wrapper which does it for you. The tremor developers appreciated these options when I created their private tap.